dotCMS/src/test/java/com/dotcms/filters/NormalizationFilterTest.java

```diff
@@ -111,5 +135,73 @@ private void compare(final String originalURI, final String expectedNormalizedUR
         assertNotNull(normalizedValueByFilter);
         assertEquals(expectedNormalizedURI, normalizedValueByFilter);
     }
+    
+    /**
+     * Test to verify the {@link NormalizationFilter} is applying properly the normalization on
+     * invalids URIs
+     */
+    @Test
+    public void test_uri_normalizer_fixes_double_slashes() throws IOException, ServletException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5436097)